backend/coins/eth/coin.go

```diff
@@ -118,6 +119,16 @@ func (coin *Coin) Unit(isFee bool) string {
 	return coin.unit
 }
 
+// SetFormatUnit implements coin.Coin.
+func (coin *Coin) SetFormatUnit(string) {
+	// No need for format unit on ETH coins.
+}
```

Comment:
I agree it is not the best to have empty functions around the code, but having `SetFormatUnit` in the interface allows to avoid casting in `set-unit` handler, keeping the code cleaner and easier to read. Let me know if you prefer to have it removed from the interface anyway :)